.github/workflows/build_workflow.yml

```
@@ -81,9 +81,7 @@ jobs:
           activate-environment: "pyremap_ci"
           miniforge-variant: Mambaforge
```

```suggestion
          miniforge-variant: latest
```
We don't want `Mambaforge` anymore.  I'm basing this on:
https://github.com/E3SM-Project/mache/blob/main/.github/workflows/build_workflow.yml#L90

---

Sorry, I missed that one in my search!

---

Oops, I clearly looked at the wrong line.  Thanks for getting it right!

---

For posterity, I found the correct variant name by looking at the code here:
https://github.com/conda-incubator/setup-miniconda/pull/360